otelcollector/deploy/addon-chart/azure-monitor-metrics-addon/templates/ama-metrics-daemonset.yaml

```diff
@@ -396,9 +396,9 @@ spec:
                 - cmd
                 - /c
                 - C:\opt\microsoft\scripts\livenessprobe.cmd
-            periodSeconds: 15
+            periodSeconds: 60
```

Comment:
kaveesh - does this interfere/stamp with any waits that we are doing in startup script for initial startup ?

Reply:
I don't think this will interfere with it. The liveness probe is set to start after a delay of 5 mins right now. Setting period seconds to 60 seconds will make it so that the probe is called every 60 seconds instead of the 15 second interval it was being called at originally. Since I was increasing the timeout duration to 60 seconds I think it makes sense to call the next liveness probe at least 60 seconds later, giving enough time for the previous one to finish/timeout.

Reply:
thanks